network-api/networkapi/tests/test_redirects.py

```diff
@@ -0,0 +1,57 @@
+from unittest import skip
+
+from django.test import RequestFactory, TestCase
+from wagtail.core.models import Site
+from wagtail_factories import SiteFactory
+
+from networkapi.utility.redirects import redirect_to_default_cms_site
```

Comment:
Considering that the function under test is located in the utility app, have you considered moving the tests into that app as well (i.e. `network-api/networkapi/utility/tests/test_redirects`)?

Reply:
Updated!